app/models/oidc_configuration.rb

```diff
@@ -64,6 +64,18 @@ def attributes
   # Always initialize a valid `config`
   after_initialize :config
 
+  # This would really be better with a XML representer
+  def to_xml(options={})
+    result = options[:builder] || ThreeScale::XML::Builder.new
+    result.oidc_configuration do |xml|
+      xml.id id
```

Comment:
If this is a new record then `id` is `nil`
Does it make sense to handle that?

Reply:
It should be an empty XML tag, no?

```xml
<id></id>
```

Reply:
Yes it is a self closing `<id/>`